src/pages/architects.js

```diff
@@ -1,16 +1,59 @@
 import React from 'react'
```

Comment:
So we can delete the architects page and leave only `directors.js` ?

Reply:
Yes we can, after commit fixes according code review